x/ibc-rate-limit/.gitignore

```diff
@@ -0,0 +1,21 @@
+# Generated by Cargo
+# will have compiled files and executables
+debug/
+target/
+
+# Generated by rust-optimizer
+artifacts/
+
+# Remove Cargo.lock from gitignore if creating an executable, leave it for libraries
+# More information here https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html
+Cargo.lock
+
+# These are backup files generated by rustfmt
+**/*.rs.bk
+
+# MSVC Windows builds of rustc generate these, which store debugging information
+*.pdb
+
+
+# Ignores local beaker state
```

Comment:
There seems to be an empty `.gitkeep` file. Do we need to keep it?

Reply:
Also part of beaker. Not sure what beaker does if the directory isn't there. @iboss-ptk is it safe to remove the empty dir?

Reply:
if it's `.gitkeep` under `contracts` dir, you can delete them, it is there for project template to generate empty `contracts` dir along.

Reply:
removed